app/actions/downloadManager.js

```diff
@@ -391,7 +392,9 @@ export function downloadPack(pack) {
         ...(thumbnailURL && { icon: 'icon.png' }),
         timePlayed: 0,
         mods: modsManifest,
-        overrideFiles: overrideFilesList
+        overrideFiles: overrideFilesList,
+        overrideArgs: DEFAULT_ARGS,
+        overrideMemory: 3096
```

Comment:
By default none of those should be present. Only when an override is set it is added. If no override is set, it is simply not present and the global is used instead